backend/src/apis/humhub/model/Profile.ts

```diff
@@ -10,11 +10,7 @@ export class Profile {
     const publishNameLogic = new PublishNameLogic(user)
     this.firstname = publishNameLogic.getFirstName(user.humhubPublishName as PublishNameType)
     this.lastname = publishNameLogic.getLastName(user.humhubPublishName as PublishNameType)
-    if (user.alias && user.alias.length > 2) {
-      this.gradido_address = CONFIG.COMMUNITY_NAME + '/' + user.alias
-    } else {
-      this.gradido_address = CONFIG.COMMUNITY_NAME + '/' + user.gradidoID
-    }
+    this.gradido_address = CONFIG.COMMUNITY_NAME + '/' + user.gradidoID
```

Comment:
Aktuell existiert m.W. keine Validierung auf einen eindeutigen Communitynamen.
Macht hier die Community-UUID nicht mehr Sinn?

Reply:
Das ist der sichtbare Link, wenn ich den auf die Community-UUID setzen muss, sieht das auf humhub ziemlich häßlich aus. Oder ich muss Humhub wieder modifzieren das der sichtbare Link sich noch mehr vom tatsächlichen Link unterscheidet was unter umständen dazu führt das der Browser die Link-Ausführung nicht zulässt.
Weil normalerweise ja meistens Pishing und Spam Links sich stark unterscheiden zwischen Namen und tatsächlicher URL.

Reply:
Wenn daraus in HumHub direkt ein Link erzeugt wird, dann müsste statt dem CommunityNamen eigentlich die Community-URL verwendet werden. Denn diese ist definitiv eindeutig und wird ja auch validiert.
Beim Community-Namen könnten ggf. auch mehrere Worte enthalten sein, die als Link gar nicht tauglich sind, oder?
Oder heißt das Attribut in HumHub einfach nur CommunityName enthält aber genaugenommen doch die CommunityURL?

Reply:
Was hältst du davon, wenn du es einfach mal ansiehst?
Logge dich in gdd.gradido.net in dein Gradido Wallet ein, gehe auf Kreise und dann weiter auf humhub und versuche jemanden über den Link unter dem Namen der bei der Benutzerübersicht angezeigt wird gradidos zu schicken.

Reply:
Mit Bernd hatten wir besprochen das wir erstmal allen Community Betreibern mündlich ans Herz legen einen Namen ohne Leerzeichen zu wählen, bis wir einen Check eingebaut haben, der sicherstellt das der gewählte Community Namen url tauglich ist.

Reply:
Wobei ein Namen mit leerzeichen durchaus funktioniert.